src/scriptworker/context.py

```diff
@@ -237,17 +241,22 @@ def event_loop(self, event_loop: asyncio.AbstractEventLoop) -> None:
     async def populate_projects(self, force: bool = False) -> None:
         """Download the ``projects.yml`` file and populate ``self.projects``.
 
-        This only sets it once, unless ``force`` is set.
+        Unless ``force`` is set or it was last fetched over an hour ago this
```

Comment:
"over an hour ago" is now a lie.  Should be something like "over PROJECTS_YML_MAX_AGE_SECONDS seconds ago".